public/fonts/vivy-icons/iconMapping.json

```diff
@@ -17,5 +17,9 @@
   "icon-will-information": "e80e",
   "icon-height-icon": "e801",
   "icon-insurance-shield": "e810",
-  "icon-phone-number": "e811"
+  "icon-phone-number": "e811",
```

Comment:
looks like e811 conflicts also with "phone-number" above, should phone-number just be removed and this used instead?

Reply:
like this?